copyright.txt

```diff
@@ -0,0 +1,3 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
```

Comment:
I'm not a huge fan of having a dedicated `copyright.txt` file besides `LICENSE` just for that.
Would it be possible to pass it as string? Or maybe simpler, we could move it into the `script` subfolder.

Reply:
Let's move it to the subfolder, the pre-commit hook is using bytes which make passing a string with line break complicated.

Reply:
https://github.com/PyCQA/astroid/pull/1441/commits/c8e8991853ab217d87f3ea4ea80e25fdd8720b30